baseplate_py_upgrader/__init__.py

```diff
@@ -45,10 +45,10 @@ def no_op_upgrade(
     "1.3": "1.4",
     "1.4": "1.5",
     "1.5": "2.0",
```

Comment:
Following up on the other convo from my PR, this I think should also be 2.5. There will be one of these migrations that makes code changes, but the rest are version bumps. With this setup, from 1.5 you'll have to run it twice, once to get to 2.0.x and again to get to 2.5.x. Since there's no other changes I don't see why we'd want to make that a two step process since there wouldn't be a significantly larger changeset to review

Reply:
The issue comes in [here](https://github.com/reddit/baseplate.py-upgrader/blob/master/baseplate_py_upgrader/__init__.py#L171), notice we grab the updater which aligns with the target version, in this case `v2.5` and runs ONLY this updater. So if we change the upgrade path off  `v1.5` from `2.0` to  `v2.5` then when a user running `v1.5` tries to update it will run the updater for `v2.5`. This is a no_op updater, it will not run the `v2.0` upgrader though, which does upgrade actions.,

Reply:
oh thats not awesome. Great catch. I had assumed that it would go through each in the list.